common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -1118,12 +1122,19 @@ def _block_matches_all(block_data):
         if 'category' in qualifiers:
             qualifiers['block_type'] = qualifiers.pop('category')
 
+        in_tree_only = kwargs.get('in_tree_only', False)
```

Comment:
instead of getting this from `kwargs`, should `in_tree_only` be specified as one of the named kwargs in this function's definition?